components/Navigation/Navigation.tsx

```diff
@@ -1,61 +1,81 @@
-import { useTranslation } from 'react-i18next';
-import cn from 'classnames';
-import { NavigationPropsI } from './NavigationProps';
-import styles from './Navigation.module.scss';
-import Link from 'next/link';
+import { useTranslation } from "react-i18next";
+import cn from "classnames";
+import { NavigationPropsI } from "./NavigationProps";
+import styles from "./Navigation.module.scss";
+import Link from "next/link";
+import { useState } from "react";
 export const Navigation: React.FC<NavigationPropsI> = ({ active }) => {
   const { t } = useTranslation();
+
+  const [isBurgerActive, setBurgerState] = useState(false);
```

Comment:
What does setBurgerState do? Sets some state, and the only state it sets is is the burger active. 
If the burger has some other states but not only is it active, you will have to rewrite this setter. It seems to be better to name the setter in more specific way.